<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AreaInfoEditor</class>
 <widget class="QDialog" name="AreaInfoEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="13" column="7">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="9" column="4">
    <widget class="QLabel" name="label">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>🚩</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="pushButtonArea">
     <property name="text">
      <string>选择区域</string>
     </property>
    </widget>
   </item>
   <item row="9" column="3">
    <widget class="QCheckBox" name="checkBoxW">
     <property name="text">
      <string>西</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>区域连接方向：</string>
     </property>
    </widget>
   </item>
   <item row="6" column="6">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>添加物件</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="checkBoxRoom">
     <property name="text">
      <string>设置为当前房间</string>
     </property>
    </widget>
   </item>
   <item row="10" column="4">
    <widget class="QCheckBox" name="checkBoxS">
     <property name="text">
      <string>南</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QCheckBox" name="checkBoxAreaLink">
     <property name="text">
      <string>连接至该区域坐标</string>
     </property>
    </widget>
   </item>
   <item row="2" column="7">
    <widget class="QSpinBox" name="spinBoxAmount">
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButtonRoom">
     <property name="text">
      <string>选择房间</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3" colspan="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>设置连接方向：</string>
     </property>
    </widget>
   </item>
   <item row="9" column="5">
    <widget class="QCheckBox" name="checkBoxE">
     <property name="text">
      <string>东</string>
     </property>
    </widget>
   </item>
   <item row="6" column="7">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>删除物件</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QLabel" name="labelRoom">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="10" column="3">
    <widget class="QCheckBox" name="checkBoxSW">
     <property name="text">
      <string>西南</string>
     </property>
    </widget>
   </item>
   <item row="8" column="5">
    <widget class="QCheckBox" name="checkBoxNE">
     <property name="text">
      <string>东北</string>
     </property>
    </widget>
   </item>
   <item row="4" column="7">
    <widget class="QSpinBox" name="spinBoxChance">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000000</number>
     </property>
     <property name="value">
      <number>1000000</number>
     </property>
    </widget>
   </item>
   <item row="8" column="4">
    <widget class="QCheckBox" name="checkBoxN">
     <property name="text">
      <string>北</string>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QCheckBox" name="checkBoxNW">
     <property name="text">
      <string>西北</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>东</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>南</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>西</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>北</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>东南</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>东北</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>西南</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>西北</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>自定义</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="10" column="5">
    <widget class="QCheckBox" name="checkBoxSE">
     <property name="text">
      <string>东南</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEditLinkDir">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="3">
    <widget class="QLabel" name="labelArea">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="6">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>生成概率：</string>
     </property>
    </widget>
   </item>
   <item row="2" column="6">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>物品数量：</string>
     </property>
    </widget>
   </item>
   <item row="7" column="6" rowspan="5" colspan="2">
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item row="4" column="3" colspan="2">
    <widget class="QLineEdit" name="lineEditCenterFlag"/>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>中心标识</string>
     </property>
    </widget>
   </item>
   <item row="13" column="6">
    <widget class="QPushButton" name="pushButton_3">
     <property name="text">
      <string>清空区域信息</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AreaInfoEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AreaInfoEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
